eng/pipelines/runtime.yml

```diff
@@ -964,6 +964,47 @@ extends:
                     eq(variables['monoContainsChange'], true),
                     eq(variables['isRollingBuild'], true))
 
+      #
+      # Android devices
+      # Build the whole product using CoreCLR and run libraries tests
+      #
+      - template: /eng/pipelines/common/platform-matrix.yml
+        parameters:
+          jobTemplate: /eng/pipelines/common/global-build-job.yml
+          helixQueuesTemplate: /eng/pipelines/libraries/helix-queues-setup.yml
+          buildConfig: Release
+          runtimeFlavor: coreclr
+          platforms:
+          - android_x64
+          - android_arm64
```

Comment:
If these indeed run on devices (see my other comment), we need to make sure we have enough devices so that we can run them on PRs by default and not block other Android jobs.

Reply:
How can we test the load?

Reply:
I think we should monitor the `Windows.11.Amd64.Android.Open` queue in Grafana to see that the waiting times don't grow so that we don't slow down `runtime` pipeline. I'm not aware of any stress we have for this.